_posts/2016-12-30-Is-not-a-Mass-is-just-a-stand-up.markdown

```diff
@@ -0,0 +1,17 @@
+---
+layout: post
+title:  "Is not a Mass, is just a stand-up"
+date:   2016-12-30 11:28:00 -0300
+categories: software, stand-up, scrum
+---
+
+Each day, at 12.00 am, the priest is ready to start the mass. It doesn’t really matter what you did or not the last day or even the quality of the code, the only most important thing is that you be there on the stand-up and talk about something.
+
+Bosses, PMs, whatever has to re-organize his priority, there are at least 10 different most important things in a project than this meeting. 
+
+> “Please inform with at least 34.3 hours before if you can’t assist a
+> stand-up”
+
+Is not a Sin and doesn’t matter the reason. 
+
+Not to be strict in this kind of thinks helps to generate a good atmosphere in the project, which in consequence helps to keep you motivated and this last adjective believe me that is something that the client is interested in.
```

Comment:
cool, yes. I have to learn to not say non negative things. I said it even in a interviews